bouquet/calculator.py

```
@@ -1,111 +1,287 @@
+        mol: Optional["Chem.Mol"] = None,
+        num_threads: int = _DEFAULT_NUM_THREADS,
+        charge: int = 0,
+        multiplicity: int = 1,
+    ) -> "Calculator":
+        try:
+            spec = _REGISTRY[method]
+        except KeyError:
+            available = ", ".join(sorted(_REGISTRY))
+            raise ValueError(
+                f"Method '{method}' is not available. Available methods: {available}"
+            )
```

Ruff (0.15.18)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)